src/main/java/com/google/sps/servlets/LoadEventServlet.java

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Query;
+import java.io.IOException;
+import java.util.logging.Logger;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/load-event")
+public class LoadEventServlet extends HttpServlet {
+
+  private static final Logger LOGGER = Logger.getLogger(LoadEventServlet.class.getName());
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+    Key keyRequested;
+    try {
+      keyRequested = getEventKey(request);
+      if (keyRequested != null) {
+        Query query = new Query("Event", keyRequested);
+
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        Entity eventRequested = datastore.prepare(query).asSingleEntity();
+        request = populateRequest(request, eventRequested);
+
+        request.getRequestDispatcher("/WEB-INF/jsp/display-event.jsp").forward(request, response);
+      }
+    } catch (Exception e) {
+      LOGGER.info("Could not retrieve event " + e);
+      request.getRequestDispatcher("/WEB-INF/jsp/event-not-found.jsp").forward(request, response);
+    }
+  }
+
+  /**
+   * Supplies the request with attributes from the event requested.
+   *
+   * @return the request with attributes set.
+   */
+  private HttpServletRequest populateRequest(HttpServletRequest request, Entity event) {
+    String name = (String) event.getProperty("eventName");
```

Comment:
Prefer `toString` instead of type conversion

Reply:
Done, thank you!